x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -313,7 +327,17 @@ export const ModelStarted = ({
           <EuiFlexItem grow>
             <EuiText color="success" size="xs">
               <h3>
-                {isCompact
+                {isSingleThreaded
```

Comment:
Nitpick(non-blocking): This is getting a bit hard to read, we should consider extracting out as a function.